.eslintrc.js

```diff
@@ -95,16 +95,7 @@ module.exports = {
         message: "Use the <VLink> component instead of <RouterLink>.",
       },
     ],
-    "no-restricted-syntax": [
-      "error",
-      ...i18nDestructureRules,
-      {
-        selector:
-          "ImportDeclaration[source.value='@vue/test-utils']:has(ImportSpecifier[local.name='shallowMount'])",
-        message:
-          "Do not use @vue/test-utils' `shallowMount`. Use @testing-library/vue's `render` instead.",
-      },
```

Comment:
This rule is moved to the unit test scope in overrides further down in this file.